### ISSUE TITLE
[Android] Cannot switch to single-tap build

### ISSUE THREAD

**Issue by jondo** [minetest/minetest#2761]
I am using Minetest 0.4.12.12 from F-Droid.
Is there a way to switch from building with double-tap to single-tap?

**Comment by Wayward1** [comment]
Unfortunately, there is not.

**Comment by MoNTE48** [comment]
The project is based on MultiCraft Minetest will support it! I'll let you know when it will be completed.

**Comment by jondo** [comment]
@MoNTE48 - why don't you just create this as a pull request to Minetest? Your [MultiCraft](https://github.com/MoNTE48/MultiCraft) project seems to just be an obsolete Minetest snapshot.

**Comment by MoNTE48** [comment]
You can see a very old version. The new version just yet published :) It will then github.com/multicraftproject

**Comment by est31** [comment]
I think ability to switch modes between build and dig should be done by 0.4.13.

**Comment by est31** [comment]
It wasn't done until now, and we are feature freezing, and this is a feature, therefore I remove it as blocker, as it can't be fulfilled.

**Comment by MoNTE48** [comment]
You can peep my code! ;)
https://github.com/MultiCraftProject/MultiCraft

**Comment by est31** [comment]
If you want us to be abled to copy your code, you could revert the change to `LGPL 3.0+`, back to 2.1+. Btw., if you have problems with sound, check out this commit: 47225346be9ab0d55bc2dbc71b6c0f085f47a75e

**Comment by MoNTE48** [comment]
I would use LGPL 2.1 and share changes and improvements. My project is a commercial, but it would not hurt the friendship MultiCraft and Minetest. But a few months ago nrz I made the most disgusting to me, so I can not change the license is now.

**Comment by est31** [comment]
That's very unfortunate to hear.

**Comment by Wayward1** [comment]
@MoNTE48, would you possibly consider making a pull request for this? IMO this is a badly needed improvement.

**Comment by paramat** [comment]
Any more comments?
Is anyone interested in coding this sometime? if not closing in 1 month.

**Comment by jondo** [comment]
The Problem will not go away by closing this issue.

**Comment by paramat** [comment]
You can't play a mobile game assuming the controls work the same as Minecraft, however, there may be other good reasons to allow this, if so please explain them.

I'm not closing the issue, please read the label, i'm bumping this for attention and looking for a coder who is interested.

**Comment by jondo** [comment]
@MoNTE48 What does "a few months ago nrz I made the most disgusting to me" mean?

**Comment by Ferk** [comment]
Why not update the license to LGPL 3.0+? I heard it has additional legal protection against bad practices (DRM and software patents), and it would allow taking code from any derived version of Minetest that uses a newer version of the LGPL.

**Comment by paramat** [comment]
Is this worth keeping open?

**Comment by Wayward1** [comment]
@paramat Definitely. As I stated above, this would be one of the biggest improvements in my opinion, and one I have been looking forward to since day one. Can you imagine having to double click on a desktop just to place a block? Especially while working on a larger-scale project? :P This would greatly improve the building speed and overall experience.

**Comment by paramat** [comment]
Ok thanks. I'm wondering why this isn't already done, is it to avoid confusion with single taps caused by dragging the view around? How do players 'dig' on Android and how is 'place' differentiated from 'dig'?

**Comment by Wayward1** [comment]
Digging is done by long-pressing on the node, and I'm unsure as to the exact reasoning behind using double taps. I don't believe there would be much of an issue with accidental placement, since dragging the view around is more of a "press, hold, and swipe" action, while an actual placement tap would be a quick "tap and release".

**Comment by paramat** [comment]
Amazing, i still think touchscreen and smartphone players are crazy, and that's 90% of server users :]

**Comment by paramat** [comment]
Any core dev support?
